src/views/canvas/CanvasSelection.js

```diff
@@ -80,6 +77,17 @@ extend(CanvasSelection.prototype, {
     })();
   },
 
+  _drawVerticalLine: function (xStart, yStart, yEnd) {
+    this.ctx.moveTo(xStart, yStart);
+    this.ctx.lineTo(xStart, yEnd);
+  },
+
+  _drawVerticalBorder: function (xPos, yStart, yEnd, hasTopBorder, hasBottomBorder) {
```

Comment:
What's the reason we don't need a `_drawHorizontalBorder` function?

Reply:
The logic for drawing horizontal borders was inline (as it was straightforward, not as complex as the logic for drawing vertical borders). However, for symmetry, let me add a `_drawHorizontalBorder` function too.